website/src/ts/components/status/status.ts

```diff
@@ -33,7 +32,7 @@ export class Status extends LitElement {
 
   override render(): TemplateResult {
     return html`
-                <sl-badge size=${this.size} variant=${this.badgeVariant}>${this.status ? this.status : 'planned'}</sl-badge>
+      <sl-badge size=${this.size} variant=${this.badgeVariant}>${this.status ? this.status : 'planned'}</sl-badge>
```

Comment:
In the component status on the website we were using before the bold version, the previous one, do we want to keep the same styling or use `subtle` from now? @DanielleRameau what do you think? :) 

<img width="1451" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/98973568-0c7a-40bc-a49a-05bc7d639c35">

Reply:
Fixed